infrastructure/main.tf

```diff
@@ -12,10 +12,10 @@ terraform {
     }
   }
   backend "s3" {
-    dynamodb_table = "ndr-terraform-locks"
-    region         = "eu-west-2"
-    key            = "ndr/terraform.tfstate"
-    encrypt        = true
+    use_lockfile = true
+    region       = "eu-west-2"
+    key          = "ndr/terraform.tfstate"
+    encrypt      = true
   }
```

Comment:
The workspace ndr108sam didn't destroy over night and I think it's because it's looking for a state-lock in Dynamo as it destroys from "main". I believe you can transition this so that it uses both lock methods. So you can keep the dynamodb_table attribute there and then remove once the state lock exists on both.